### PR TITLE
Doc: Adding RailShuttle to physical_mode list

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

**Comment by TeXitoi** [comment]
And which picto I put in navitia-playground for that? How use that?

File: documentation/ntfs/ntfs_changelog.md [hunk 1]
```diff
@@ -43,3 +43,4 @@
     * ajout des temps d'embarquement et de débarquement dans le fichier stop_times.txt.
 * Version 0.6.3 du 25/01/2017
     * ajout du champ dataset_extrapolation dans le fichier datasets.txt
+* Version 0.6.4 du 31/03/2017
```
Comment (pbougue) [inline documentation/ntfs/ntfs_changelog.md:46]: missing the change itself, no?

Comment (pbougue) [reply documentation/ntfs/ntfs_changelog.md:46]: (side idea) And maybe we should follow the "breaking.feature.bugfix" versionning thing from : semver.org

Comment (prhod) [reply documentation/ntfs/ntfs_changelog.md:46]: changelog fixed. I used the 3rd identifier because there is no real change in the spec

**Review by pbougue — CHANGES_REQUESTED** [review]
Looks like you commited outdated submodules files (and http url), is it intentional?

**Comment by prhod** [comment]
Sorry for the miss, I fixed the PR

**Review by pbougue — APPROVED** [review]

**Pull request merged**